src/shaderlib/fp64/math-fp64.glsl

```diff
@@ -247,16 +249,17 @@ vec2 exp_fp64(vec2 a) {
   p = mul_fp64(p, r);
   t = mul_fp64(p, INVERSE_FACTORIAL_5_FP64);
 
-  s = sum_fp64(s, t);
-  p = mul_fp64(p, r);
-  t = mul_fp64(p, INVERSE_FACTORIAL_6_FP64);
+  // s = sum_fp64(s, t);
```

Comment:
Drop the comments below or add another comment line in front of each comment indicating why we are keeping them.